hack/deploy-wait.sh

```diff
@@ -20,6 +20,6 @@ done
 
 if ! $ready; then 
     echo "Timed out waiting for features to be ready"
-    oc get nodes
+    kubectl get nodes
```

Comment:
nit -
CLI ?= kubectl

probably a better name than CLI... but you get the idea

Reply:
using the `OPERATOR_EXEC` variable, as it already exists. Does it sound good?